srv/components/system/network/prepare.sls

```diff
@@ -43,18 +43,20 @@ Stop and disable NetworkManager service:
 # If not killed explicitly, it causes network restart to fail: COSTOR-439
 Kill dhclient:
   cmd.run:
-    - name: pkill -SIGTERM dhclient
+    # - name: pkill -SIGTERM dhclient
+    - name: dhclient -x
```

Comment:
@ypise  as long as we will start dhclient with particular iface don't you think we need to specify iface here as well (not sure how self kill relates with that actually) ?

Reply:
We probably don't need an interface here, but I'm wondering what happens if there are one than one processes running.

Reply:
Agreed, we need to only change the value for one interface, but it doesn't work.
So we need to go back to pkill.
I'll prefer to disable the kill and restart logic as it doesn't make sense anymore.